rllib/core/models/base.py

```diff
@@ -30,8 +30,15 @@ class ModelConfig(abc.ABC):
     It is therefore a means of configuration for RLModules. However, ModelConfigs are
     not restricted to be used only with Catalog or RLModules.
     A usage Example together with a Model can be found in the Model.
+
+    Args:
+        input_dims: The input dimensions of the network
+        output_dims: The output dimensions of the network.
     """
 
+    input_dims: Union[List, Tuple] = None
+    output_dims: Union[List, Tuple] = None
```

Comment:
List or Tuple of what? Ints?

Reply:
Also when would u use multiple output dims?

Reply:
As per offline discussion: We'll keep this generic with `output_dims` instead of `output_dim`, maybe even extend this to nested structures later.